scipy/stats/tests/test_continuous_basic.py

```diff
@@ -662,3 +664,40 @@ def test_methods_with_lists(method, distname, args):
                             [f(*v) for v in zip(x, *shape2, loc, scale)],
                             rtol=1e-15, atol=1e-15)
 
+
+@pytest.mark.parametrize('distname, args', distcont)
+def test_pickle_frozen(distname, args):
```

Comment:
These tests are good and seem to be more extensive then  `test_pickling` at https://github.com/scipy/scipy/blob/master/scipy/stats/tests/test_distributions.py#L2752
It's be also good to consolidate and remove duplication.

Reply:
My function is probably duplicating [check_pickling in common_tests](https://github.com/scipy/scipy/blob/master/scipy/stats/tests/common_tests.py#L273). The stats test suite is so large that sometimes it's difficult to know what's out there, it's so easy to overlook something. I might remove the test I added, I'm not sure it is better than that in common_tests, and the original test also tests discrete.

Reply:
What check_pickling doesn't do is test if the frozen distribution pickle/unpickles. The link you provided in [test_distributions](https://github.com/scipy/scipy/blob/master/scipy/stats/tests/test_distributions.py#L2752) does do that, but only for 3 distributions. I might amend `check_pickling` to test them all.

Reply:
By a3ea650cd6240aad3ed321766b7253ac17ecc6a8 check_pickling also tests pickling of all frozen distributions.